packages/fxa-auth-server/lib/senders/emails/fluent-localizer.ts

```diff
@@ -78,7 +78,10 @@ class FluentLocalizer {
     context = { ...context, ...context.templateValues };
     if (template !== '_storybook') {
       // TODO: #11471 Improve dynamically rendered actions & subjects in email.js, etc.
-      if (template === 'postRemoveTwoStepAuthentication') {
+      if (
+        template === 'postRemoveTwoStepAuthentication' ||
+        template === 'verifyLoginCode'
```

Comment:
We got a little lucky here that we could just stick `-line` like this huh 😅 I'm glad we'll be addressing #11471 soon enough.